tests/functional/testplan/runners/pools/test_pool_process.py

```diff
@@ -17,7 +17,7 @@
 from .func_pool_base_tasks import schedule_tests_to_pool
 
 
-pytestmark = pytest.mark.skipif(True, reason='Process Pool tests are unstable')
+# pytestmark = pytest.mark.skipif(True, reason='Process Pool tests are unstable')
```

Comment:
Same as above - did you mean to leave this line commented out? I think we will need more testing before we can say the pool tests are stable.

Reply:
This hasn't been resolved either?